### DISCUSSION TITLE
[Question][Hw5] What is difference between two tables 'X'

### DISCUSSION THREAD

**Discussion by nubi12336** [prosyslab-classroom/cs348-information-security#625] [Q&A]
In the abstract definition of Analysis.run, 
![image](https://user-images.githubusercontent.com/79691513/120491046-e1eff200-c3f3-11eb-9130-b0e6c4167aa0.png)
I think left 'X' and right 'X' are different because if not, left 'X' is a table that all the labels have the bottom memories, so that join with right X's will become a bottom(empty) memory.
However, I don't know what the difference is. Can you explain? Right 'X' is updated by some process?

**Comment by KihongHeo** [comment]
That is just an equation that specifies the characteristics of the abstract semantics. 
Instead, solving the equation is a different story. Equation itself does not involve any computation, but equation solving does.
The equation can be solved by the fixed point algorithm introduced in the lecture.

**Comment by nubi12336** [comment]
Can you tell me which lecture should I take a look at..?

**Comment by ee12ha0220** [comment]
It is in the second lecture of week 11. To be specific, the expression can be found in page 25 of week 11-2 pdf file.
